tools/tests/bundle-ignore-files.js

```diff
@@ -12,4 +12,10 @@ selftest.define("bundle-ignore-files", () => {
   ];
 
   matchingInputs.forEach(input => selftest.expectEqual(patterns.some(p => p.test(input)),true));
+
+  const nonMatchingInputs = [
+    '/imports/components/Icon/index.js',
```

Comment:
[This](http://superuser.com/questions/298785/icon-file-on-os-x-desktop) states that the file is actually called `Icon\r`. May need to update that regex again ;)